packages/create-plugin/templates/common/tests/smoke.spec.ts

```diff
@@ -0,0 +1,5 @@
+import { test, expect } from '@grafana/plugin-e2e';
+
+test('should load dashboard page', async ({ dashboardPage }) => {
+  await expect(dashboardPage.ctx.page).toHaveTitle('New dashboard - Dashboards - Grafana');
+});
```

Comment:
Should this just be something like:

```suggestion
test('should load dashboard page', async ({ page }) => {
  await expect(page).toHaveTitle('New dashboard - Dashboards - Grafana');
});
```

Reply:
Because we do not navigate to any other page than the home page, right?

Reply:
This test is now replaced with plugin type specific tests. 
https://github.com/grafana/plugin-tools/pull/847/commits/0c09a476976a59a48507bc9db712c2ed356ec62a